.buildkite/pipeline.yml

```
@@ -67,6 +67,16 @@ steps:
         agents:
           slurm_gpus: 1
 
+      - label: "Unit: distributed cuda spaces"
+        key: "gpu_distributed_extruded_cuda_spaces"
+        command:
+          - "julia --project -e 'using CUDA; CUDA.versioninfo()'"
+          - "srun julia --color=yes --check-bounds=yes --project=test test/Spaces/distributed_cuda/space_construction.jl CUDA"
+        agents:
+          slurm_gpus: 1
+          slurm_nodes: 3
+          slurm_tasks_per_node: 1
```

Not sure: maybe it can't get scheduled?

Do they need to be on distinct nodes? If not then try
```suggestion
          slurm_gpus_per_task: 1
          slurm_ntasks: 3
```